kolibri/auth/permissions/base.py

```diff
@@ -92,42 +92,22 @@ def _get_target_object(self, obj):
             return getattr(obj, self.target_field)
 
     def user_can_create_object(self, user, obj):
-
         roles = getattr(self, "can_be_created_by", None)
-
-        assert isinstance(roles, tuple), \
-            "If `can_be_created_by` is None, then `user_can_create_object` method must be overridden with custom behavior."
```

Comment:
It is these asserts that causes the incompatibility issue with other permission class.